### PR TITLE
Realgud updates

### PR DESCRIPTION
**Pull request by rocky**
From the `realgud` recipe we add:
 
* `ipdb`

From the `realgud recipe` we remove and put into a separate repository called `realgud-old-debuggers`:

* `pydb`  
* `pydbr`
* `trepan8`
* `trepanx`

We also add create new recipes for:

* Ruby byebug `realgud-byebug`
* Ruby pry `realgud-pry`
* Ruby debugger2  `realgud-ruby-debugger2`

### CODE REVIEW

**Comment by purcell** [comment]
Thanks. In the cases where you have `;; Package-Requires: ((realgud))`, you'll need to add the version number of the dependency, which is mandatory, e.g. `;; Package-Requires: ((realgud "1.2"))`.

**Comment by rocky** [comment]
@purcell I think I have all of the respective package on github now having proper `Package-Requires`.  Specifically: 

* https://github.com/rocky/realgud-old-debuggers/commit/5aa477e2dcf8b029d2c4ceae8514a5d975f5bf67
* https://github.com/rocky/realgud-pry/commit/7c9f37fd1daef70bec52726d5a7bb0c52997af91
* https://github.com/rocky/realgud-byebug/commit/518027e20ce9478ffcbaaae5309477a5b70f1a08

Thanks for all your help. I'm sorry it has been this much trouble. (I make mistake; that is why I work with debuggers) 

With luck, this is the last of the fixes.

**Comment by rocky** [comment]
Actually, in that list above I forgot realgud-debugger2. I'm working on that now...

**Comment by rocky** [comment]
And with https://github.com/rocky/realgud-ruby-debugger2/commit/52062bdd3914cba8e1adcb364abcbbb45de8bfc3 I think that now is all of the debuggers.

File: recipes/realgud-debugger2 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(realgud-debugger2 :fetcher github :repo "rocky/realgud-ruby-debugger2"
```
Comment (purcell) [inline recipes/realgud-debugger2]: Here the `realgud-debugger2` name doesn't match the name of `realgud-rdb2.el` -- the package name and main contained filename should match, so you probably want to rename this package file.

Inside `realgud-rdb2.el` has `(defgroup realgud ...)` in it, which should presumably be `(defgroup realgud-rdb2 ...)`

File: recipes/realgud-old-debuggers [hunk 2]
```diff
@@ -0,0 +1,6 @@
+(realgud-old-debuggers :fetcher github :repo "rocky/realgud-old-debuggers"
+	 :files ("realgud-old-debuggers.el"
```
Comment (purcell) [inline recipes/realgud-old-debuggers]: Same problem here with the `(defgroup realgud ...)`.

File: recipes/realgud-pry [hunk 3]
```diff
@@ -0,0 +1,3 @@
+(realgud-pry :fetcher github :repo "rocky/realgud-pry"
+	 :files ("realgud-pry.el"
```
Comment (purcell) [inline recipes/realgud-pry]: Same here with the `(defgroup realgud ...)`.

File: recipes/realgud-debugger2 (hunk 1 above)
Comment (rocky) [inline recipes/realgud-debugger2]: Thanks. Should now be fixed by [5c6dc28]

Comment (rocky) [inline recipes/realgud-debugger2]: Thanks. Now fixed in [5c6dc28]

File: recipes/realgud-old-debuggers (hunk 2 above)
Comment (rocky) [inline recipes/realgud-old-debuggers]: Fixed I think by https://github.com/rocky/realgud-old-debuggers/commit/eee3d9d88bfe94d21f08716217184e44a4161e55

File: recipes/realgud-pry (hunk 3 above)
Comment (rocky) [inline recipes/realgud-pry]: Fix now by https://github.com/rocky/realgud-pry/commit/60c24ab619fba3adc4aac6ba6cf73fdb4e11e696  I think.

**Comment by purcell** [comment]
Thanks -- I've squashed these commits together and merged them. :-)

**Comment by rocky** [comment]
Thanks for handling all of this.  My capacity for making mistakes though is
unbounded. There was a typo in old-debuggers. So yet another pull request.

Wondering if this will unsquash those squash commits? If so, should I try
to unsquash?

**Pull request closed without merging**